cmd/protoc-gen-go-tableau-checker/hub.go

```diff
@@ -197,7 +185,7 @@ func (h *Hub) checkCompatibility(newHub *tableau.Hub, protoPackage string, break
 func (h *Hub) Check(dir string, format format.Format, options ...Option) error {
 	opts := ParseOptions(options...)
 	// load hub
-	loadErr := h.load(dir, format, options...)
+	loadErr := h.load(loadTypeDefault, opts.ProtoPackage, dir, format, opts.LoadOptions...)
 	if loadErr != nil && !opts.SkipLoadErrors {
 		return loadErr
```

Comment:
We should distinguish load error and check error by https://pkg.go.dev/errors#example-Is

- For load error, use `xerrors.NewDesc(err)` to get pretty-print error message.
- For check error, simple print `err.Error()`